client/lib/post-normalizer/index.js

```diff
@@ -469,6 +471,32 @@ normalizePost.withContentDOM = function( transforms ) {
 
 normalizePost.content = {
 
+	createContentWithLinebreakElementsOnly: function createContentWithLinebreakElementsOnly( post, callback ) {
+		if ( ! post.__contentDOM ) {
```

Comment:
this doesn't need to be a content transform if we're not using the contentDOM spun up by the wrapper